validphys2/src/validphys/deltachi2.py

```diff
@@ -219,9 +219,7 @@ def draw(self, pdf, grid, flstate):
 
         errorstd = stats.std_error()
         # color cycle iterable
-        pcycler = ax._get_lines.prop_cycler
-        next_prop = next(pcycler)
-        color = next_prop["color"]
+        color = ax._get_lines.get_next_color()
```

Comment:
`get_next_color` is indeed the best way to iterate on the colors. 

Maybe for absolute consistency (and this is very minor and perhaps not necessary) the only leftover place in which this should be reflected is in the example: https://github.com/NNPDF/nnpdf/blob/f553482181a9d785a2386ae29c44efc765b74d7f/validphys2/examples/API_example.ipynb#L326-L327.